test_db_handler.py::test_insert_files_db fails with mongoDB error

Unit test `tests/unit_tests/test_db_handler.py::test_insert_files_db` fails occasionally with the following error message:

```
        file_id = db.insertFileToDB(fileName, "sandbox")
>       assert file_id == db._getFileMongoDB("sandbox", "test_file.dat")._id
E       AssertionError: assert <gridfs.grid_file.GridOut object at 0x7fcabd2ce100> == ObjectId('635401432d0a2a03d3f81222')
E        +  where ObjectId('635401432d0a2a03d3f81222') = <gridfs.grid_file.GridOut object at 0x7fcabd2ce3d0>._id
E        +    where <gridfs.grid_file.GridOut object at 0x7fcabd2ce3d0> = <function DatabaseHandler._getFileMongoDB at 0x7fcac20adb80>('sandbox', 'test_file.dat')
E        +      where <function DatabaseHandler._getFileMongoDB at 0x7fcac20adb80> = <simtools.db_handler.DatabaseHandler object at 0x7fcabd2d0130>._getFileMongoDB
```

Note entirely clear to me what the issue is.

--- a/tests/unit_tests/test_db_handler.py
+++ b/tests/unit_tests/test_db_handler.py
@@ -34,6 +34,15 @@ def db_cleanup(db, random_id):
     db.dbClient["sandbox"]["metadata_" + random_id].drop()
 
 
+@pytest.fixture()
+def db_cleanup_file_sandbox(db):
+    yield
+    # Cleanup
+    logger.info("Dropping the temporary files in the sandbox")
+    db.dbClient["sandbox"]["fs.chunks"].drop()
+    db.dbClient["sandbox"]["fs.files"].drop()
+
+
 def test_reading_db_lst(db):
 
     logger.info("----Testing reading LST-----")
@@ -316,7 +325,7 @@ def test_separating_get_and_write(db):
         assert io.getOutputFile(fileNow, dirType="model", test=True).exists()
 
 
-def test_insert_files_db(db):
+def test_insert_files_db(db, db_cleanup_file_sandbox):
 
     logger.info("----Testing inserting files to the DB-----")
     logger.info(
@@ -328,10 +337,6 @@ def test_insert_files_db(db):
 
     file_id = db.insertFileToDB(fileName, "sandbox")
     assert file_id == db._getFileMongoDB("sandbox", "test_file.dat")._id
-
-    logger.info("Dropping the temporary files in the sandbox")
-    db.dbClient["sandbox"]["fs.chunks"].drop()
-    db.dbClient["sandbox"]["fs.files"].drop()
 
 
 def test_get_all_versions(db):

